.azure-pipelines.yml

```
@@ -79,6 +79,8 @@ jobs:
   steps:
   - template: tools/ci/azure/checkout.yml
   - script: |
+      set -euo pipefail
+      git fetch origin
```

Maybe
```suggestion
      git fetch --depth 50 --quiet origin master
```
to avoid fetching everything (albeit it seems to only take <30s anyway) and make the output less verbose